source/_includes/nav-tabs-pattern.html

```diff
@@ -14,3 +14,21 @@
   {% if include.usabilitytab != false %}<li role="presentation"><a href="#usability-test" aria-controls="usability-test" role="tab" data-toggle="tab">Usability Test</a></li>{% endif %}
 -->
 </ul>
+<script>
+    $(function() {
+        // Workaround for bug with hidden C3 charts in v0.4.11.
+        var resize = function () {
+            setTimeout(function () {
+                if (window.dispatchEvent) {
+                    window.dispatchEvent(new Event('resize'));
+                }
+                // Special case for IE
+                if ($(document).fireEvent) {
+                    $(document).fireEvent('onresize');
+                }
+            }, 100);
+        }
+        $(document).on('shown.bs.tab', 'a[data-toggle="tab"]', resize);
+        $(document).ready(resize);
```

Comment:
Yes because the user can refresh the web browser. In that case, the previously hidden chart will look bad.